src/app/shared/models/DtoResponse/series/serie-meta.model.ts

```diff
@@ -0,0 +1,7 @@
+import { SerieMeta } from './SerieMeta.model';
+
+export class SerieMetaResponseDto {
+    status: boolean;
+    message: string;
+    content: SerieMeta;
```

Comment:
In english, series always has a "s" at the end

Reply:
I know, but in the entire application, "serie" is used (instead of "series")